src/Services/privateApiService.js

```diff
@@ -6,14 +6,24 @@ const config = {
     }
 }
 
+const privateInstance = axios.create({
+    baseURL: 'http://ongapi.alkemy.org/public/api',
+});
+
+privateInstance.interceptors.request.use((config)=> {
+    config.headers.Authorization = localStorage.getItem('token');
+    config.headers.Group = 91;
+    return config
+})
+
+export const privateGet = async (url,id,params={}) => {privateInstance.get(`${url}${id?`/${id}`:''}`,{params})};
```

Comment:
the id can be null, you should intercept that adding ? on the parameters. Sometimes its good to have all on the same line and sometimes not, this is a place where its not. Maybe a constant would help some programmers to understand better this functionallity
```
=> {
    const idPlaceholder = id ? `/${id}` : '';
    privateInstance.get(`${url}${idPlaceholder}`, {params})
};
```

Reply:
Ok, I already changed it